database/frontend/animations.json

```diff
@@ -2,39 +2,54 @@
   {
     "id": "23",
     "name": "Lottie files",
-    "description": "LottieFiles takes away the complexity from Motion Design. It lets you Create, Edit, Test, Collaborate and Ship a Lottie in the easiest way possible.",
+    "description": {
+      "en": "LottieFiles takes away the complexity from Motion Design. It lets you Create, Edit, Test, Collaborate and Ship a Lottie in the easiest way possible.",
+      "es": "LottieFiles te quita la complejidad del Motion Design. Te permite Crear, Editar, Probar, Colaborar y Enviar un Lottie de la manera más fácil posible."
+    },
     "url": "https://lottiefiles.com/",
     "category": "frontend",
     "subcategory": "animations"
   },
   {
     "id": "27",
     "name": "Cubic Bezier",
-    "description": "A tool for visualisation and creation of cubic-bezier curves for animation in CSS",
+    "description": {
+      "en": "A tool for visualisation and creation of cubic-bezier curves for animation in CSS",
+      "es": "Una herramienta para la visualización y creación de curvas cúbicas para la animación en CSS"
+    },
     "url": "https://cubic-bezier.com/",
     "category": "frontend",
     "subcategory": "animations"
   },
   {
     "id": "28",
```

Comment:
Also i see everything on the webpage is changed manually. It will be best to automate this instead of changing manually and should support different languages not just two as said early.  
Doing this manually will really not be efficient because each time a link is accepted to the hub, converting manually each time won't be ideal.